H2MLauncher.UI/App.xaml.cs

```diff
@@ -127,21 +113,32 @@ private static IConfigurationRoot BuildConfiguration()
                 .AddJsonStream(input);
 
             // NOTE: using two builders because we use the embedded resource as a stream
-            IConfigurationRoot root = builder.Build();
+            var defaultConfiguration = builder.Build();
```

Comment:
what happened here? does it return a `IConfigurationRoot` or `IConfiguration` or `Configuration` ? I have no clue!

Reply:
😂 will fix